src/main/java/com/connexta/store/adaptors/StoreStatus.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.store.adaptors;
+
+public class StoreStatus {
```

Comment:
✏️ The only one who should ever use these strings is the class that calls updateStatus. I suggest moving these to that class.